package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "lint": "stylelint content/styles/**/*.css --config postcss.linter.json",
     "dev": "postcss ./content/styles/styleguide.css -o ./content/dist/styleguide.css --env dev",
-    "watch": "npm run lint && postcss ./content/styles/styleguide.css -o ./content/dist/styleguide.css --env dev --watch --verbose",
+    "watch": "postcss ./content/styles/styleguide.css -o ./content/dist/styleguide.css --env dev --watch --verbose",
```

Comment:
Why not the following?
```
"watch": "npm run dev -- --watch --verbose",
```

Reply:
better! addressed with 9e877fe298df0563ec20b37e61181cf6a4034f27